src/simplewallet/simplewallet.cpp

```diff
@@ -281,6 +281,16 @@ namespace
   {
     return boost::lexical_cast<std::string>(version >> 16) + "." + boost::lexical_cast<std::string>(version & 0xffff);
   }
+
+  crypto::secret_key decrypt_key(const crypto::secret_key &key, const std::string &passphrase)
```

Comment:
The same code is also in `wallet2.cpp` which seems redundant to me. Shouldn't these functions `decrypt_key  / encrypt_key` be defined in a common place, e.g. in `cryotonote_format_util.cpp` since their functionalities seem rather generic?